courses/serializers.py

```diff
@@ -500,3 +502,133 @@ def get_certificate(self, user_program_enrollment):
         """
         certificate = user_program_enrollment.get("certificate")
         return ProgramCertificateSerializer(certificate).data if certificate else None
+
+
+class ProgramRequirementDataSerializer(StrictFieldsSerializer):
```

Comment:
Are there any caveats that would limit the future reuse of these serializers when displaying program requirements to learners?  I'm guessing no, but figured I might be overlooking something and now is a good time to ask.

Reply:
I don't think we'd want to because the data structure is a little more complicated than it needs to be since it's how `MP_Node.dump_bulk` operates.